spec/javascripts/PaymentGateways/braintree/BraintreeForm.spec.jsx

```diff
@@ -1,9 +1,24 @@
+// @flow
```

Comment:
Are this changes related to updating packages? Maybe they belong to another PR (refactor)

Reply:
The upgrade make these tests fail, not sure why, but I think we should merge it together.

Reply:
OK, let's merge